winterfell/webapp/src/styles/claimStart.styl

```diff
@@ -1,22 +1,89 @@
 @import "standards.styl"
 
+maxContentWidthPx = 560px
+iconWrapperPx = 54px
+
 #claim-start-view
-  width: maxLayoutWidthPx
-  margin: largeSpacing auto
-  padding: largeSpacing megaSpacing
-  box-sizing: border-box
+  min-height: 600px
```

Comment:
why do we need min height? could we make it in units percent or `vh` instead of pixels if we do need it?